### PR TITLE
add definition for intel/2016b toolchain

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2146.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/febda9611bd05358cec8d6bb75fffbdf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2598.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/9f4c710197a73eaedbd546f9f932f7fa for a full test report.

File: easybuild/easyconfigs/i/imkl/imkl-11.3.3.210-iimpi-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+sources = ['l_mkl_%(version)s.tgz']
+checksums = ['f72546df27f5ebb0941b5d21fd804e34']
+
+dontcreateinstalldir = 'True'
+
+license_file = HOME + '/licenses/intel/license.lic'
+
+interfaces = True
+
```
Comment (wpoely86) [inline easybuild/easyconfigs/i/imkl/imkl-11.3.3.210-iimpi-2016b.eb]:
```
components = ['intel-mpi', 'intel-psxe', 'intel-imb']
```

Comment (wpoely86) [inline easybuild/easyconfigs/i/imkl/imkl-11.3.3.210-iimpi-2016b.eb]: I mean:
```
components = ['intel-mkl']
```

File: easybuild/easyconfigs/i/iimpi/iimpi-2016b.eb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+description = """Intel C/C++ and Fortran compilers, alongside Intel MPI."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+compver = '2016.3.210'
+suff = '-GCC-5.4.0-2.26'
+dependencies = [
+    ('icc', compver, suff),
+    ('ifort', compver, suff),
+    ('impi', '5.1.3.181', '', ('iccifort', '%s%s' % (compver, suff))),
```
Comment (wpoely86) [inline easybuild/easyconfigs/i/iimpi/iimpi-2016b.eb]: add to `impi`:
```
components = ['intel-mpi', 'intel-psxe', 'intel-imb']
```

File: easybuild/easyconfigs/i/imkl/imkl-11.3.3.210-iimpi-2016b.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/i/imkl/imkl-11.3.3.210-iimpi-2016b.eb]: done

File: easybuild/easyconfigs/i/iimpi/iimpi-2016b.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/i/iimpi/iimpi-2016b.eb]: done

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 5 (5 easyconfigs in this PR)
node2090.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/cae425376f69f779b972f8e231097696 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 5 (5 easyconfigs in this PR)
node2412.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/322882f3f75f47ef244e3fb86fec2f9e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2605.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f7bc6b684af736bba39e4508d5189317 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2160.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e70fae71371dd88f489c02ca43fd1eff for a full test report.

**Comment by boegel** [comment]
Going in, thanks for the review @wpoely86!

**Pull request merged**